### DISCUSSION TITLE
Problem during installation

### DISCUSSION THREAD

**Discussion by Prosolit-Pierre** [RehanSaeed/Schema.NET#8] [Q&A]
Hi,

I'm trying to add your library to my project (in c# 4.0). Unfortunately, I have an error message during the installation : 

`Could not install package You are trying to install this package into a project that targets '.NETFramework,Version=v4.0', but the package does not contain any assembly references or content files that are compatible with that framework`

I've tried with a project in v4.5 but i have the same error (except the version).  Is it possible to use your library with a project in .net 4.0 version ? 

Regards,

Pierre.

**Comment by RehanSaeed** [comment]
Try 4.5.2. Anything less than that version is not actually supported by Microsoft.
